datalake/src/main/java/com/sequenceiq/datalake/service/sdx/attach/SdxDetachException.java

```diff
@@ -0,0 +1,7 @@
+package com.sequenceiq.datalake.service.sdx.attach;
+
+public class SdxDetachException extends RuntimeException {
```

Comment:
nit: this could extend `CloudbreakServiceException`

Reply:
Done.